src/contracts/BPool.sol

```diff
@@ -455,4 +457,6 @@ contract BPool is BToken, BMath, IBPool {
   function _burnPoolShare(uint256 amount) internal {
     _burn(address(this), amount);
   }
+
+  function _afterFinalize() internal virtual {}
```

Comment:
should we add a unit test that checks this internal virtual method is being called?